### DISCUSSION TITLE
CUY UNIVERSITY-SEMESTER 4 | Page Populer Not Found

### DISCUSSION THREAD

**Discussion by rifkimaulanaa** [deaafrizal/forum#10] [Troubleshooting Room] [answered]
Izin bang kenalin nama gw rifki maulana gw mahasiswa semester 1 yang memperdalami JS dan gw mengalami problem mas pada playlist semester 4 yang di mana populer nya notfound berikut file page .jsx nya di folder app dan file populer

![code-snapshot](https://github.com/deaaprizal/forum/assets/146139031/1b936f5f-ae8c-4824-a3d9-f1530acde9ff)
![populer](https://github.com/deaaprizal/forum/assets/146139031/be7586e0-6d7b-4364-904d-a8d610c268c4)

**Comment by deaafrizal** [comment]
boleh liat juga folder structure nya? dan juga .env file nya bro.

yang error page not found itu ketika akses url apa?

**Reply by rifkimaulanaa** [reply]
api nya aman mas cuman erornya pas nge akses  http://localhost:3000/populer 
![Screenshot (226)](https://github.com/deaaprizal/forum/assets/146139031/f7a9d514-8ebf-4948-9f33-73e518323be8)
![Screenshot folder](https://github.com/deaaprizal/forum/assets/146139031/deae4624-d66a-41c3-b330-2fa2591fa85d)

**Reply by deaafrizal** [reply, marked as answer]
kayanya page.jsx deh gak pake "s", bukan pages.jsx

**Reply by rifkimaulanaa** [reply]
thankyou mas udah resolved

**Reply by deaafrizal** [reply]
oraitss nice!
